cmake/Modules/Packages.cmake

```diff
@@ -40,31 +40,39 @@ if(PYTHON_EXECUTABLE)
         ${PYTHON_ROOT_DIR}/include)
 endif()
 
-
 ################################################################################
 #
-#        MKL (required)
+#        OpenCV (required for CPU acceleration)
 #
 ################################################################################
```

Comment:
This OpenCV and MKL section were swapped in your branch. Reswap them so that the git diff is cleaner (to avoid unnecessary changes upstream).